tst/testinstall/help.tst

```diff
@@ -0,0 +1,9 @@
+# Test help
+
+gap> f := function()
+> ?help
+Syntax error: '?' cannot be used in this context in stream:2
+?help
+    ^
+Syntax error: end expected in stream:3
+^
```

Comment:
Before this PR:
```
gap> 1 + ?foobasrsd
Syntax error: expression expected
1 + ?foobasrsd
    ^
>
```
With this PR:
```
gap> 1 + ?foobasrsd
Syntax error: literal expected
1 + ?foobasrsd
             ^
>
```
That's not tragic, but I kind of prefer the first error. Then again, this is hopefully so rare that it won't matter, ever :-).

On the other hand, old version:
```
gap> repeat  ?foobarsdf;  until true;
Syntax error: until expected
repeat  ?foobarsdf;  until true;
        ^
```
With this PR:
```
gap> repeat  ?foobarsdf;  until true;
Syntax error: '?' cannot be used in this context
repeat  ?foobarsdf;  until true;
                               ^
> 
```
That text is more helpful than the old (but the wrong place is pointed out, which isn't quite as cool).